docs/item_template.md

```diff
@@ -2594,7 +2594,7 @@ The honor rank the player needs to have to use this item.
 
 ### RequiredCityRank
 
-`field-no-description|22`
+Its use is unknown. All items have 0.
```

Comment:
have you double checked this by querying something like

```sql
SELECT DISTINCT(RequiredCityRank) FROM `item_template`;
```

?

Reply:
![immagine](https://user-images.githubusercontent.com/519778/64069503-a3d80400-cc4a-11e9-91ba-3ca0a4ce306b.png)

Yes